wayland-logout.1

```diff
@@ -0,0 +1,41 @@
+.Dd $Mdocdate: November 27 2020 $
+.Dt WAYLAND-LOGOUT 1
+.Os
+.Sh NAME
+.Nm wayland-logout
+.Nd generic logout command for wayland compositors
+.Sh DESCRIPTION
+.Nm
+is a compositor agnostic command for simulating a
+logout with any wayland complaint compositor.
+.Pp
+It sends a
+.Sy SIGINT
+.Xr signal 3
+to the running compositor that is obtained by looking
+at the
+.Sx environment
+variables.
+.Sh ENVIRONMENT
+The command uses the
+.Ev WAYLAND_DISPLAY
+and
+.Ev XDG_RUNTIME_DIR
+.Nm
+environment variables to get the
+.Xr socket 2
+corresponding to the compositor instance.
+.Sh EXIT STATUS
+The
+.Nm
+command exits with 1 if an error has occurred along with
+the error reasoning and exits with 0 on success.
+.Sh EXAMPLES
+To kill the current compositor instance, run inside
+the compositor:
+.Dl $ wayland-logout
+.Pp
+A typical use case, to kill a specific compositor
+instance of another user (with uid=1002), run as
+the other user (or as root):
```

Comment:
```suggestion
another user:
```

Reply:
hmm, this doesn't flow right. 
We already referred to the user with uid=1002, it doesn't make sense to call another user again.